src/Util/Emailer.php

```diff
@@ -64,7 +64,7 @@ public function __construct(IConfig $config, IPConfig $pConfig, App\BaseURL $bas
 		$this->l10n        = $defaultLang;
 
 		$this->siteEmailAddress = $this->config->get('config', 'sender_email');
-		if (empty($sysEmailAddress)) {
+		if (empty($this->siteEmailAddress)) {
```

Comment:
Hard to find when PHP is not declarative. Good catch!

Reply:
What do you mean "not declarative"? Static analyzers would have caught this.

Reply:
If PHP would be a declarative (paradigm) language, `$sysEmailAddress` cannot be used here.